javaslang-benchmark/pom.xml

```diff
@@ -27,6 +27,16 @@
             <artifactId>javaslang</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <!--<dependency>-->
+        <!--<groupId>org.scala-lang</groupId>-->
+        <!--<artifactId>scala-library</artifactId>-->
+        <!--<version>2.12.0-M4</version>-->
+        <!--</dependency>-->
```

Comment:
@paplorinc yes, Ruslan is right. Let's remove this block. When Scala 2.12 is out we will update our dependencies.

Btw - awesome work!! :-)